dev/run-tests.py

```diff
@@ -559,6 +559,10 @@ def main():
 
         changed_modules = test_modules
         if len(changed_modules) == 0:
+            # If SKIP_PACKAGING is explicitly set (no matter true or false),
+            # let run_python_packaging_tests to determine whether it should be run.
+            if not os.environ.get("SKIP_PYTHON") and os.environ.get("SKIP_PACKAGING") is not None:
+                run_python_packaging_tests()
```

Comment:
Hm, to run pip packaging test, Spark has to be built first but here is before we build. I think it won't work correctly?

Reply:
oops, let me double check, thanks!